tests/MockInjectorTest.php

```diff
@@ -9,10 +9,16 @@
 use Prophecy\Prophecy\ObjectProphecy;
 use Tests\Dummy\DummyDependency;
 use Tests\Dummy\DummySubDependency;
+use PHPUnit\Framework\Attributes\CoversClass;
+use PHPUnit\Framework\Attributes\CoversMethod;
 
-/**
- * @coversDefaultClass \Bigcommerce\MockInjector\MockInjector
- */
+#[CoversClass(MockInjector::class)]
+#[CoversMethod(MockInjector::class, 'create')]
+#[CoversMethod(MockInjector::class, 'invoke')]
+#[CoversMethod(MockInjector::class, 'checkPredictions')]
+#[CoversMethod(MockInjector::class, 'getAllMocks')]
+#[CoversMethod(MockInjector::class, 'getMock')]
+#[CoversMethod(MockInjector::class, '__construct')]
```

Comment:
Any reason why you've opted to push `CoversMethod` from individual tests to the top level annotation? This changes the behaviour slightly, where `testCreate()` for example will contribute to the coverage of `__construct()` without actually performing any assertions on the outcome.